dpnp/backend/kernels/dpnp_krnl_random.cpp

```diff
@@ -435,7 +435,8 @@ void dpnp_rng_multinomial_c(
     }
     else
     {
-        std::int32_t* result1 = reinterpret_cast<std::int32_t*>(result);
+        DPNPC_ptr_adapter<std::int32_t> result_ptr(result, size, true, true);
```

Comment:
Parameter `target_no_sycl` is `true`. Do you think it's correct value for the parameter?

Reply:
Parameter `target_no_sycl` with `false` value for result_ptr leads to _Segmentation fault_ when calling `viRngMultinomial()`.